src/dashboard/Push/PushDetails.react.js

```diff
@@ -39,7 +39,13 @@ const EXP_STATS_URL = 'http://docs.parseplatform.org/ios/guide/#push-experiments
 let getMessage = (payload) => {
   if(payload) {
     let payloadJSON = JSON.parse(payload);
-    return payloadJSON.alert ? payloadJSON.alert : payload;
+		if (payloadJSON.alert.body) {
```

Comment:
@natanrolnik,

Sorry I took so long to circle back, but I've been tied up. I was able to modify the component to display both the title and the body as part of detail by checking the existance of an tyeof objective to keep consistant as per your observation. Here is a screenshot.
<img width="1280" alt="screen shot 2017-08-09 at 8 29 55 am" src="https://user-images.githubusercontent.com/20729160/29130029-823badf6-7cdd-11e7-854d-cd06c61a62f1.png">